source/mainnet/smart-contracts/guides/integration-test-contract.rst

```diff
@@ -108,8 +111,10 @@ Deploying smart contract modules is a two-step process.
 First, you load the module with the function |module_load_v1|_, then you deploy it to the chain with the method |Chain_module_deploy_v1|_.
 Loading as a separate step allows you to reuse the loaded module across multiple tests for efficiency.
 
-The module to load should be a ``wasm`` module compiled with ``cargo concordium build``.
-For example, for ``cargo concordium build --embed-schema --out my_module.wasm.v1``, you write:
+The module to load should be a ``wasm`` module compiled with ``cargo concordium build`` or ``cargo concordium test --out path/to/wasm/module``.
```

Comment:
_Should_ it be wasm or _must_ it be wasm? I don't like should because it indicates room for doubt.